routers/movies-router.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const {
+  getMovies,
+  getById,
+  createMovie,
+  upsertMovie,
+  modifyMovie,
+  deleteMovie,
+} = require('../controllers/routers-controller')
+
+const moviesRouter = new express.Router()
+
+moviesRouter.get('/', getMovies)
+moviesRouter.get('/:id', getById)
+moviesRouter.post('/', createMovie)
+moviesRouter.put('/', upsertMovie)
+moviesRouter.patch('/:id', modifyMovie)
+moviesRouter.delete('/:id', deleteMovie)
+
+module.exports = { moviesRouter }
```

Comment:
You can use any name you want even when you export an object, for example:
```js
const { funcName } = require('./md')
```

-->
```js
const { funcName: anyNameYouWant } = require('./md')
```

I guess it's a matter of personal preference, and because you already discussed about import/exports only using default exports, I will change my code to align with it where I can (sometimes I can't do it because I need to export multiple functions/objects..)